crates/stages/src/test_utils/macros.rs

```diff
@@ -97,7 +97,7 @@ macro_rules! stage_test_suite {
 
             // Run stage unwind
             let unwind_input = crate::stage::UnwindInput {
-                unwind_to: stage_progress, stage_progress, bad_block: None,
+                unwind_to: stage_progress, stage_progress: previous_stage, bad_block: None,
```

Comment:
Was this a bug? Cc @rkrasiuk

Reply:
Yes, it was, but in the way that tests were set it didn't show.

Reply:
@rakita thanks for spotting & fixing

Reply:
moved change here: https://github.com/paradigmxyz/reth/pull/895